src/Test/L0/Listener/Configuration/Testable/TestWindowsServiceControlManager.cs

```diff
@@ -0,0 +1,28 @@
+using System.ServiceProcess;
+
+using Microsoft.VisualStudio.Services.Agent.Listener.Configuration;
+
+namespace Test.L0.Listener.Configuration.Testable
+{
+    public class TestWindowsServiceControlManager : WindowsServiceControlManager
+    {
+        public ServiceController SelectedServiceController { get; set; }
+
+        protected override void InstallService(
```

Comment:
This is only specific windowsServiceControlManager, If I have to keep it in the interface I had to implement in linux and osx service control manager as well which will have no implementation..